dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpConfigConstants.java

```diff
@@ -405,6 +405,9 @@ private HttpConfigConstants() {
     // This turns on the ability to limit window_update frames to be sent only when 1/2 the window is gone.
```

Comment:
Can you just clean up the `Contributors:` in the copyright header since we're touching this file?